src/operator/constants.tsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react'
+import {CellInfo, ResourceInfo} from 'src/types/operator'
+import {Link} from 'react-router-dom'
+
+export const accountHeaderInfo = [
+  'Company Name',
+  'Acct ID',
+  'Owner Email',
+  'Users',
+  'Balance',
+  'Account Type',
+  'Billing Provider',
+]
+
+export const accountColumnInfo: CellInfo[] = [
+  {
+    path: 'billingContact.companyName',
+    name: 'company-name',
+    defaultValue: '',
+  },
+  {
+    path: 'id',
+    name: 'account-id',
+    defaultValue: '',
+    renderValue: value => (
+      <Link to={`/operator/accounts/${value}`}>{value}</Link>
+    ),
+  },
+  {
+    path: 'billingContact.email',
+    name: 'email',
+    defaultValue: '',
+  },
+  {
+    path: 'users.length',
+    name: 'users',
+    defaultValue: '',
+  },
+  {
+    path: 'balance',
+    name: 'balance',
+    defaultValue: '',
+  },
+  {
+    path: 'type',
+    name: 'acct-type',
+    defaultValue: '',
+  },
+  {
+    path: 'marketplace.shortName',
+    name: 'marketplace',
+    defaultValue: 'Zuora',
+  },
+]
+
+export const organizationColumnHeaders = [
+  'Org Name',
+  'Org ID',
+  'Owner Email',
+  'Account ID',
+  'Balance',
+  'Type',
+  'Provider',
+  'Region',
+  'Date Created',
+]
+
+export const organizationColumnInfo: CellInfo[] = [
+  {
+    path: 'name',
+    name: 'org-name',
+    defaultValue: '',
+  },
+  {
+    path: 'idpeID',
+    name: 'org-id',
+    defaultValue: '',
+    renderValue: value => (
+      <Link to={`/operator/organizations/${value}`}>{value}</Link>
+    ),
+  },
+  {
+    path: 'account.billingContact.email',
+    name: 'email',
+    defaultValue: '',
+  },
+  {
+    path: 'account.id',
+    name: 'account-id',
+    defaultValue: '',
+  },
+  {
+    path: 'account.balance',
+    name: 'acct-balance',
+    defaultValue: '',
+  },
+  {
+    path: 'account.type',
+    name: 'acct-type',
+    defaultValue: '',
+  },
+  {
+    path: 'provider',
+    name: 'provider',
+    defaultValue: '',
+  },
+  {
+    path: 'region',
+    name: 'region',
+    defaultValue: '',
+  },
+  {
+    path: 'date',
+    name: 'date',
+    defaultValue: '',
+  },
+]
+
+export const userColumnInfo: CellInfo[] = [
```

Comment:
This one looks like leftover cruft from the partially-implemented Users page we had in here at one point. Optional whether we keep it or delete it until we bring back the Users page.